kahuna/public/js/search/results.js

```diff
@@ -316,11 +316,11 @@ results.controller('SearchResultsCtrl', [
             if (angular.isUndefined(orderBy)) {
                 orderBy = $stateParams.orderBy;
             }
-
             return mediaApi.search($stateParams.query, angular.extend({
                 ids:        $stateParams.ids,
                 archived:   $stateParams.archived,
                 free:       $stateParams.nonFree === 'true' ? undefined : true,
+                isDeleted:  $stateParams.isDeleted === 'true' ? true : false,
```

Comment:
I don't think we need the ternary here? `$stateParams.isDeleted === 'true'` should be enough.